### DISCUSSION TITLE
Moving database Plex from Docker to Plex LXC

### DISCUSSION THREAD

**Discussion by defeatedbythecat** [tteck/Proxmox#1762] [Help Needed] [answered]
I already have Plex in a docker container in another Vm, but rather than start over, I'd like to migrate the database. It's easy enough for me to get the database and associated files from Docker, but with the LXC I don't know where to start?

Can you point me in the right direction?

**Comment by tteck** [comment, marked as answer]
If you need to know the data path within the Plex LXC, it is located at `/var/lib/plexmediaserver/Library/`.

**Reply by defeatedbythecat** [reply]
Thank you, I was poking around and didn't get there!

**Comment by AlexGeek77** [comment]
I just moved from a Synology install over to a privileged LXC on my cluster. I followed the instructions on the Plex site here and had no troubles. 
https://support.plex.tv/articles/201370363-move-an-install-to-another-system/

**Reply by defeatedbythecat** [reply]
Thanks for that link. now I know where, that'll help me migrate!

**Comment by tteck** [comment]
Once Plex is in a LXC, moving data around to another Plex LXC is made simple by a copy-data [script](https://raw.githubusercontent.com/tteck/Proxmox/main/misc/copy-data/plex-copy-data-plex.sh)

https://github.com/tteck/Proxmox/blob/main/misc/copy-data/README.md

**Reply by david-a-kelly** [reply]
![image](https://github.com/tteck/Proxmox/assets/93332205/e192f6e7-95bf-4975-b599-bf851077e1f1)

This script looks to be my saviour for a dead plex LXC, but I've got this fairly non-descript error - any ideas?

p.s. thanks for all your AMAZING work!
